<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>voirPlusVisiteur</class>
 <widget class="QDialog" name="voirPlusVisiteur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color:#00A086;
border:none;
border-radius:10px;</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nom du visiteur :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nomVisiteurlbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Adresse : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="adresselbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Provenance : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="provenancelbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>233</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Nombre Total des sites visit√©s : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nbsitelbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>188</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Effectif total des visiteurs : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="effectVisiteurlbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>661</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
